tests/algorithms/test_intensity.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import pytest
+
+import reciprocalspaceship as rs
+
+
+@pytest.mark.parametrize("inplace", [True, False])
+@pytest.mark.parametrize("output_columns", [None, ("I_backcalc", "SigI_backcalc")])
+def test_compute_intensity_from_structurefactor(ref_hewl, inplace, output_columns):
```

Comment:
Nice test -- if we decide to keep the `dropna` argument, please add a test (or parametrize this test) to hit that part of the function